src/shared-components/SearchBar/SearchBar.module.css

```
@@ -0,0 +1,228 @@
+.searchBarContainer {
+  display: flex;
+  align-items: center;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "align-items" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)